lib/lune.js

```diff
@@ -304,7 +304,7 @@ function phase_hunt (sdate) {
   }
 
   let adate = new Date(sdate.getTime() - (45 * 86400000)) // 45 days prior
-  let k1 = Math.floor(12.3685 * (adate.getFullYear() + ((1.0 / 12.0) * adate.getMonth()) - 1900))
+  let k1 = Math.floor(12.3685 * (adate.getFullYear() + (1.0 / 12.0) * adate.getMonth() - 1900))
```

Comment:
I copied it from `phase_hunt`. I don't know what it means either. :(

(The comment prefacing `meanphase()` shows where it comes from but not why it exists or what it means!)